### DISCUSSION TITLE
How is IBorder supposed to be used

### DISCUSSION THREAD

**Discussion by danielftz** [dotnet/maui#4822] [Q&A] [answered]
I have some confusion regarding _IBorder_ and _Border_ usage. I understand from the MSDN docs that _Border_ is supposed to be used as a parent object almost in the same way as _Frame._ However I would like to use IBorder which I was told would render border for any class that implements it like this:

```
public class MyEntry: Entry, IBorder
{
       public IBorderStroke Border => new Border()
       {
           Stroke = new SolidColorBrush(Colors.Red),
           StrokeShape = new RoundRectangle()
           {
   	        CornerRadius = new CornerRadius(15, 15, 15, 15),
           },
           StrokeThickness = 4,
       };
 
       public MyEntry()
       {
       }
}
```
Context: https://github.com/dotnet/maui/issues/3951#issuecomment-1044989248

However it doesn't seem to work, and I have no idea if the error is on my part or not. Some help would be greatly appreciated.

**Comment by danielftz** [comment, marked as answer]
@Clancey Could you please help me out with this?

**Reply by Clancey** [reply]
Which platform?

**Reply by danielftz** [reply]
Android 11

**Reply by Clancey** [reply]
It's a bug, I figured it out on the plane, I need to port the fix to main. You are going to need to wait until p14  😞

**Reply by danielftz** [reply]
Thank you. Much to look forward to in P14 then.

**Reply by Clancey** [reply]
https://github.com/dotnet/maui/pull/4909

**Comment by daniel-smith-sonepar** [comment]
@danielftz - did you get this working?  if so do you have a full sample?  I am trying to do the same thing and the border is not being rendered on iOS.
